VueApp/src/CTS/pages/ManageBundles.vue

```diff
@@ -32,7 +31,7 @@
     const loading = ref(false)
 
     async function load() {
-        get(apiUrl + "cts/bundles?milestone=false").then(r => bundles.value = r.result)
```

Comment:
@bsedwards I noticed that milestone bundles were suppressed from showing up even if they were created. Was this intentional?